src/Command/Elastic/CreateDynamicStatsTemplateCommand.php

```diff
@@ -0,0 +1,129 @@
+<?php
+/**
+ * @file
+ * Command to create an dynamic index template elasticsearch.
+ */
+
+namespace App\Elastic\Command;
+
+use Elasticsearch\Client;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * Class CreateDynamicStatsTemplateCommand.
+ */
+class CreateDynamicStatsTemplateCommand extends Command
+{
+    private $client;
+
+    protected static $defaultName = 'app:elastic:create-stats-template';
+
+    /**
+     * ElasticStatsTemplateCommand constructor.
+     *
+     * @param Client $client
+     *   ElasticSearch Client
+     */
+    public function __construct(Client $client)
+    {
+        parent::__construct();
+
+        $this->client = $client;
+    }
+
+    /** {@inheritdoc} */
+    protected function configure()
+    {
+        $this->setDescription('Create dynamic stats index template. Command is idempotent and can safely be called multiple times.');
+    }
+
+    /** {@inheritdoc} */
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        try {
+            $result = $this->createStatsTemplate();
+
+            if ($result['acknowledged']) {
+                $io->success('Dynamic index template created.');
+
+                return 0;
+            }
+
+            $io->error('Unknown error when creating template');
+
+            return 1;
+        } catch (\Exception $exception) {
+            $io->error($exception->getMessage());
+
+            return 1;
+        }
+    }
+
+    /**
+     * Create a dynamic index template to ensure proper indexing in ElasticSearch.
+     *
+     * @see https://www.elastic.co/guide/en/elasticsearch/reference/6.8/indices-templates.html
+     * @see https://www.elastic.co/guide/en/elasticsearch/reference/6.8/dynamic-templates.html
+     *
+     * @return array
+     */
+    private function createStatsTemplate(): array
+    {
+        $params = [
+            'name' => 'statistics_index_template',
+            'body' => [
+                'index_patterns' => [
+                    'stats_*',
```

Comment:
I remember that the stats pattern comes from .env maybe use that pattern?

Reply:
Done